pinot-connectors/pinot-flink-connector/src/main/java/org/apache/pinot/connector/flink/http/HttpClient.java

```diff
@@ -0,0 +1,130 @@
+/**
```

Comment:
It's used in the quickstart to show how to use REST API to fetch the schema and table config to config the connector. Otherwise developer needs to manually craft those configs. 

Also, you can take a look at `PinotConnectionUtils` for the useful config management.

Reply:
yes, but it is not related to the flink connector. 

for the sake of this PR's scope. i would suggest we drop it and add in separate PR.

mainly for the following reasons. 
- it doesn't relate to flink connector, at first I thought we are utilizing http to directly ingest into pinot via some sort of REST API. but if it is not the case it shouldn't be part of flink-pinot connector but rather in pinot-tools. 
- @xiangfu0 also think we can refactor this part out later. doesn't make sense to get this in an refactor it out if it is not part of the critical path
- there's already one util in test call ControllerUtils that can be used for demonstration purpose.

Not saying it is not useful, but it would be much easier to address it separately.

Reply:
Not really. I think this is an integral part to show how the connector is used as in `QuickStart`, without it I don't think the connector is complete. It doesn't make sense to expect developers to manually create the Schema as well as TableConfig, not to mention several configurations decoration needed in the `PinotConnectionUtils `.

Xiang's point is that in later PR we can refactor this part of code with a reusable client lib. However, from the connector's user perspective, it doesn't change the way of using the connector. 

Makes sense?

Reply:
hmm , I see. This file is filtered by one of the .gitignore rules. Renamed it and pushed again.